DataFormats/MuonReco/interface/Muon.h

```diff
@@ -183,6 +183,45 @@ namespace reco {
 			   RPCHitAndTrackArbitration, GEMSegmentAndTrackArbitration, ME0SegmentAndTrackArbitration };
     
     ///
+    /// ====================== STANDARD SELECTORS ===========================
+    ///
+    enum Selector {
+      CutBasedIdLoose        = 1UL<< 0,  
+      CutBasedIdMedium       = 1UL<< 1,  
+      CutBasedIdMediumPrompt = 1UL<< 2,  // medium with IP cuts
+      CutBasedIdTight        = 1UL<< 3,  
+      CutBasedIdGlobalHighPt = 1UL<< 4,  // high pt muon for Z',W' (better momentum resolution)
+      CutBasedIdTrkHighPt    = 1UL<< 5,  // high pt muon for boosted Z (better efficiency)
+      PFIsoVeryLoose         = 1UL<< 6,  // reliso<0.40
+      PFIsoLoose             = 1UL<< 7,  // reliso<0.25
+      PFIsoMedium            = 1UL<< 8,  // reliso<0.20
+      PFIsoTight             = 1UL<< 9,  // reliso<0.15
+      PFIsoVeryTight         = 1UL<<10,  // reliso<0.10
+      TkIsoLoose             = 1UL<<11,  // reliso<0.10
+      TkIsoTight             = 1UL<<12,  // reliso<0.05
+      SoftCutBasedId         = 1UL<<13,  
+      SoftMvaId              = 1UL<<14,  
+      MvaLoose               = 1UL<<15,  
+      MvaMedium              = 1UL<<16,  
+      MvaTight               = 1UL<<17,
+      MiniIsoLoose           = 1UL<<18,  // reliso<0.40
+      MiniIsoMedium          = 1UL<<19,  // reliso<0.20
+      MiniIsoTight           = 1UL<<20,  // reliso<0.10
+      MiniIsoVeryTight       = 1UL<<21   // reliso<0.05
+
+    };
+    
+    bool passed( unsigned int selection ) const { return (selectors_ & selection)==selection; }
+    unsigned int getSelectionMask() const { return selectors_; }
```

Comment:
there are no accessors starting with "get" in this class.
Better keep the style and call this e.g. ```selectionMask()```

Reply:
also, there is some naming inconsistency ```selectors_``` vs ```selectionMask```.
better use one to reduce the number of variants to remember in possible future updates

Reply:
fixed